### DISCUSSION TITLE
Equivalent to Alacritty SimpleFullscreen

### DISCUSSION THREAD

**Discussion by lsig** [ghostty-org/ghostty#3214] [Feature Requests, Ideas] [answered]
I was wondering if I could do something like this in ghostty:
```bash
# Alacritty setup
startup_mode = "SimpleFullscreen"
```
This makes the terminal to feel like it is in full screen, no macos menubar on top but also allows the terminal to be transparent.

This is my current ghostty setup for full screen:
```bash
# Fullscreen settings
macos-titlebar-style = "hidden"
macos-non-native-fullscreen = true
fullscreen = true
```

Edit**:

I managed to get it to work when opening a new window but not on startup:
```bash
macos-titlebar-style = "hidden"
window-decoration = false
macos-non-native-fullscreen = true
fullscreen = true
```

if you could help it would be greatly appreciated :))

Alacritty:
![image](https://github.com/user-attachments/assets/83949486-4cda-4ff4-ba2c-f6a7d2b435d2)

Ghostty:
![image](https://github.com/user-attachments/assets/fe7695a8-dc1b-4e97-9abb-8b3bddd60032)

**Comment by mitchellh** [comment, marked as answer]
Accepted #8388
